hal/src/main/native/include/hal/UsageReporting.h

```diff
@@ -140,7 +160,7 @@ namespace nUsageReporting
     } tInstances;
 
     /**
-     * Reports the usage of a resource of interest.
+     * Report the usage of a resource of interest.
```

Comment:
I think this was correct before.

"The code reports the" vs "The code report the"

Reply:
I agree, but that's what the file now has upstream, apparently.

Reply:
It was changed by @ThadHouse in #1132, so let's retain the change rather than reverting to upstream.

Reply:
I didn't realize that PR changed up streams. We need to revert that back

Reply:
We copy this file every new Ni image so it never should have been changed

Reply:
If we want it to be verbatim upstream, I should put the trailing whitespace I found back in too.

Reply:
It gets copied by the Ni update script so yes. It should not be modified.

Reply:
Yeah let's keep it as close as possible to what the new upstream one will ultimately be.